features/attached_commands.feature

```diff
@@ -24,6 +47,44 @@ Feature: Command behaviour when attached to an UA subscription
            | hirsute |
            | groovy  |
 
+    @series.xenial
+    @series.bionic
+    @uses.config.machine_type.lxd.vm
+    Scenario Outline: Attached refresh config in a ubuntu machine vm
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        When I attach `contract_token` with sudo
+        When I append the following on uaclient config
+            """
+            ua_config:
+                http_proxy: "http://localhost:1234"
+                https_proxy: "http://localhost:12345"
+            """
+        When I run `ua refresh config` with sudo
+        Then I will see the following on stdout:
+            """
+            Setting snap proxy
+            Setting Livepatch proxy
+            Successfully processed your ua configuration.
+            """
+        When I append the following on uaclient config
+            """
+            ua_config:
+                http_proxy: ""
+                https_proxy: ""
+            """
+        And I run `ua refresh config` with sudo
+        Then I will see the following on stdout:
+            """
+            No proxy set in config; however, proxy is configured for: snap, livepatch.
+            See https://discourse.ubuntu.com/t/ubuntu-advantage-client/21788 for more information on ua proxy configuration.
+
+            Successfully processed your ua configuration.
+            """
```

Comment:
@lucasmoura @mitechie @blackboxsw - let me know what you think of the new output in this scenario

Also, is this the correct link where we will be adding the proxy related documentation?

Reply:
+1, we should probably have a header for proxy doc material in that link but just that link will work for now.